src/components/addnewbook.js

```diff
@@ -1,15 +1,17 @@
 import { useDispatch } from 'react-redux';
 import { uuid } from 'uuidv4';
 import { addBook } from '../redux/books/books';
+import { addTostore } from '../utils/bookstoreapi';
 
 const AddNewBook = () => {
   const dispatch = useDispatch();
-  const submitBook = () => {
+  const submitBook = async () => {
     const newBook = {
       id: uuid(),
       category: document.getElementById('category').value,
       title: document.getElementById('bookname').value,
     };
+    await addTostore(newBook.id, newBook.title, newBook.category);
     dispatch(addBook(newBook));
   };
   return (
```

Comment:
- [ ] Let us make sure that we are adding the valid data to be rendered in the component, kindly make our input as required before we add a book or make sure that we are not adding empty data.
 
![arr23](https://user-images.githubusercontent.com/55635977/132039195-5b9c01ef-4d22-41b2-badb-5b4f7ac5d931.png)

- After adding a book, our input field should be empty instead of keeping the added book in the field.

![arr24](https://user-images.githubusercontent.com/55635977/132039201-d4bda3ec-ce14-49bd-b0c0-c5f724006bbf.png)